src/styles/blocks/product.scss

```diff
@@ -0,0 +1,81 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: $product-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__disc {
+    font-weight: $font-weight-med;
+    font-size: $primary-sm-font;
+    padding: 40px 0 4px;
+    line-height: 18px;
```

Comment:
You may combine for font-weight, font-size, line-height in a place holder. for reusability. This is for all font styling used in various places.